Wrappers/Python/cil/framework/framework.py

```diff
@@ -524,6 +528,11 @@ def calculate_magnification(self):
         :rtype: list
         '''
         raise NotImplementedError
+  
+    def is_simple(self):
+        r'''Returns `True` if the the geometry matches the default definitions with no offsets or rotations
```

Comment:
update the docstring to what it really does.
Probably `description` is more appropriate as it does not return only a boolean.